src/utils/getEmptyCells.tsx

```diff
@@ -0,0 +1,15 @@
+import type { CellsType } from '../components/types/GameType';
+
+const getEmptyCellsIndex = (cells: CellsType): [number, number][] => {
+  const emptyCells: [number, number][] = [];
+  cells.forEach((row, rowIndex) => {
+    row.forEach((value, colIndex) => {
+      if (value === null) {
+        emptyCells.push([rowIndex, colIndex]);
+      }
+    });
+  });
+  return emptyCells;
```

Comment:
사실 이렇게 함수로 깔끔하게 분리까지 되어 있으면 상관없긴 한데.. 그냥 제가 `push` `forEach` 를 좀 안 좋아해서 마이너한 리뷰 하나 남깁니다 ㅋㅋㅋㅋ 반영 안 해주셔도 괜찮아요

```suggestion
  return cells.flatMap((row, i) => row.flatMap((cell, j): [number, number][] => cell === null ? [[i, j]] : []));
```